src/raylib/model/Animation.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+** EPITECH PROJECT, 2022
+** Bomberman-Global-Offensive
+** File description:
+** Animation
+*/
+
+#include "Animation.hpp"
+
+namespace raylib
+{
+    namespace model
+    {
+        Animation::Animation(const std::string_view animPath)
```

Comment:
the `const` keyword on pass-by-copy parameter is useless, since you cannot modify the caller's version anyway